app/Http/Controllers/Api/V1/ProjectController.php

```diff
@@ -53,50 +54,33 @@ public function __construct(ProjectRepositoryInterface $projectRepository)
     }
 
     /**
-     * Get All Projects
+     * Get All Projects of login user
      *
      * Get a list of the projects of a user.
-     * @urlParam suborganization required The Id of a suborganization Example: 1
+     * @param Request $request
```

Comment:
Need to add docs as per following conventions 
https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api#specifying-request-parameters

Also need to move "* @param Request $request" etc 
just above "* @return Response"
consult  toggleStarter()" for example.